pkgs/development/mobile/androidenv/ndk-bundle/default.nix

```diff
@@ -7,13 +7,33 @@ let
     coreutils file findutils gawk gnugrep gnused jdk python3 which
   ]) + ":${platform-tools}/platform-tools";
 in
-deployAndroidPackage {
+deployAndroidPackage rec {
   inherit package os;
-  nativeBuildInputs = [ makeWrapper ]
-    ++ lib.optionals stdenv.isLinux [ autoPatchelfHook ];
+  nativeBuildInputs = [ makeWrapper autoPatchelfHook ];
```

Comment:
autoPatchelfHook fails on mac, it has to stay behind the isLinux optional flag.

Reply:
Ah, forgot to undo that.